.env.example

```
@@ -25,6 +25,10 @@ DB_CONN_MAX_LIFETIME=5m
 GOOGLE_APPLICATION_CREDENTIALS=./path/to/credentials.json
 GOOGLE_SERVICE_ACCOUNT={"type":"service_account","project_id":""}
 
+LANGFUSE_SECRET_KEY=sk-lf-your-langfuse-key
+LANGFUSE_PUBLIC_KEY=pk-lf-your-langfuse-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGFUSE_PUBLIC_KEY key should go before the LANGFUSE_SECRET_KEY key

(UnorderedKey)